tern/command_lib/base.yml

```diff
@@ -379,3 +379,41 @@ npm:
           - "pkgs=`npm list -g | sed 's/\\(^[^a-z]*\\)\\([_.a-z0-9-]*\\)\\(@.*\\)/\\2/g' | grep -v '/usr*' | uniq`"
           - "for p in $pkgs; do npm view $p repository.url; done"
     delimiter: "\n"
+# golang----------------------------------------------------------------------
+go:
+  pkg_format: ''
+  os_guess:
+    - 'None'
+  path:
+    - 'usr/local/go/bin'
+  pkg_separators:
+    - '@'
+    - '@^'
+    - '@~'
+  pinning_separator: ':'
+  names:
+    invoke:
+      1:  
+        container:
+          - "/usr/local/go/bin/go list -m all | tail -n +2 | cut -d ' ' -f 1" 
```

Comment:
Since https://github.com/tern-tools/tern/pull/780 will now set the PATH when it exists, we no longer need to call the go command using `/usr/local/go/bin/go`. We can just use `go` now.